products/bigtable/api.yaml

```diff
@@ -0,0 +1,76 @@
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: Bigtable
+display_name: BigTable
+versions:
+  - !ruby/object:Api::Product::Version
+    name: ga
+    base_url: https://bigtableadmin.googleapis.com/v2/
+scopes:
+  - https://www.googleapis.com/auth/bigtable
+apis_required:
+  - !ruby/object:Api::Product::ApiReference
+    name: BigTable API
+    url: https://console.cloud.google.com/apis/library/bigtable.googleapis.com/
+objects:
+  - !ruby/object:Api::Resource
+    name: 'AppProfile'
+    kind: 'bigtable#appProfile'
+    base_url: projects/{{project}}/instances/{{instance}}/appProfiles/{{app_profile_id}}
+    create_url: projects/{{project}}/instances/{{instance}}/appProfiles?appProfileId={{app_profile_id}}
+    update_url: projects/{{project}}/instances/{{instance}}/appProfiles?updateMask=
+    update_verb: :PATCH
+    delete_url: projects/{{project}}/instances/{{instance}}/appProfiles/{{app_profile_id}}?ignoreWarnings=true
```

Comment:
As above this can be specified in `parameters`. Also I don't think we should default to ignoring warnings.